Installing-Oppia-using-Docker.md

```diff
@@ -1,4 +1,5 @@
 This guide provides step-by-step instructions for installing Oppia using Docker. Docker simplifies the installation process and ensures a consistent environment across different systems, making it easier to set up and run Oppia.
+If you get stuck somewhere and need help, please refer to [Getting Help Page](https://github.com/oppia/oppia/wiki/Get-help)
```

Comment:
Ditto: please use MediaWiki syntax for internal links. I'll update the style guide to specify this